Sources/Utils/POSIX/Stringify.cpp

```diff
@@ -170,6 +171,26 @@ char const *Stringify::Errno(int error) {
   }
 }
 
+char const *Stringify::WaitStatus(int status) {
+  ATT_TLS static std::string result;
+
+  if (WIFEXITED(status)) {
+    result = "exited, status=" + ToString(WEXITSTATUS(status));
+  } else if (WIFSIGNALED(status)) {
+    result = "killed by signal " + ToString(Signal(WTERMSIG(status)));
```

Comment:
I don't think you're supposed to use `ToString` here. Also, you might want to do `Stringify::Signal(...)` instead of just `Signal(...)` to make it a bit easier to understand here.